aspnetcore/blazor/hybrid/static-files.md

```diff
@@ -1,60 +1,57 @@
+This article describes how to consume static asset files in Blazor Hybrid apps.
 
 [!INCLUDE[](~/blazor/includes/blazor-hybrid-preview-notice.md)]
 
 In a Blazor Hybrid app, static files are *app resources*, accessed by Razor components using the following approaches:
 
-* [.NET MAUI](#net-maui): [:::no-loc text="Xamarin.Essentials":::](/xamarin/essentials/)
+* [.NET MAUI](#net-maui): [:::no-loc text=".NET MAUI Essentials":::](https://devblogs.microsoft.com/xamarin/xamarin-essentials-1-7-and-introducing-net-maui-essentials/#net-maui-essentials)
 * [WPF](#wpf) and [Windows Forms](#windows-forms): <xref:System.Resources.ResourceManager>
 
-When static assets are only used in the Razor components, they can be consumed by Razor components from the web root (`wwwroot` folder) in a similar way to Blazor WebAssembly and Blazor Server apps. For more information, see the [Static assets limited to Razor components](#static-assets-limited-to-razor-components) section.
+When static assets are only used in the Razor components, static assets can be consumed from the web root (`wwwroot` folder) in a similar way to Blazor WebAssembly and Blazor Server apps. For more information, see the [Static assets limited to Razor components](#static-assets-limited-to-razor-components) section.
 
 ## .NET MAUI
 
-In .NET MAUI apps, the `MauiAsset` build action and [:::no-loc text="Xamarin.Essentials":::](/xamarin/essentials/) are used for static assets.
+In .NET MAUI apps, [*raw assets*](/dotnet/maui/fundamentals/single-project#raw-assets) using the `MauiAsset` build action and :::no-loc text=".NET MAUI Essentials"::: are used for static assets.
 
 Place raw assets into the `Resources/Raw` folder of the app. The example in this section uses a static text file.
 
 `Resources/Raw/Data.txt`:
 
 ```text
-This is some text from a static text file resource.
+This is text from a static text file resource.
 ```
 
-> [!NOTE]
-> When a file is added to the `Resources/Raw` folder in Visual Studio, the file is automatically assigned the `MauiAsset` build action.
-
 The following Razor component:
 
-* Calls <xref:Xamarin.Essentials.FileSystem.OpenAppPackageFileAsync%2A> to obtain a <xref:System.IO.Stream> for the resource.
-* Establishes a new <xref:System.IO.StreamReader> for the resource.
+* Calls `Microsoft.Maui.Storage.FileSystem.OpenAppPackageFileAsync` to obtain a <xref:System.IO.Stream> for the resource.
+* Reads the <xref:System.IO.Stream> with a <xref:System.IO.StreamReader>.
 * Calls <xref:System.IO.StreamReader.ReadToEndAsync%2A?displayProperty=nameWithType> to read the file.
 
 `Pages/StaticAssetExample.razor`:
 
 ```razor
 @page "/static-asset-example"
 @using System.IO
-@using Microsoft.Maui.Storage
 @using Microsoft.Extensions.Logging
+@using Microsoft.Maui.Storage
 @inject ILogger<StaticAssetExample> Logger
 
 <h1>Static Asset Example</h1>
 
-<p>@@dataResourceText: @dataResourceText</p>
+<p>@dataResourceText</p>
```

Comment:
I'm fine with leaving this as before (with `@@`). Just wasn't familiar with the fact that was used elsewhere in the docs 😄

Reply:
Ahhhh ... I don't think it matters. Let's just go with this. *Fewer bits on the wire when the doc is requested!* 😆